src/components/ui/tooltip.tsx

```
@@ -0,0 +1,31 @@
+"use client";
+
+import { cn } from "@/lib/utils";
+import * as TooltipPrimitive from "@radix-ui/react-tooltip";
```

ESLint

[error] Expected "@radix-ui/react-tooltip" (value-external) to come before "@/lib/utils" (value-internal).

(perfectionist/sort-imports)